bindata/v4.0.0/service-serving-cert-signer-controller/ns.yaml

```diff
@@ -1,8 +0,0 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: openshift-service-ca
-  labels:
-    openshift.io/run-level: "1"
-  annotations:
-    openshift.io/node-selector: ""
```

Comment:
where is this gone?

Reply:
All 3 controllers had the same content for `ns.yaml` and even though I renamed the cert signing path git chose to use the one from the apiserver injector.